.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+
+*       @panther-labs/detections @panther-labs/threat-research
```

Comment:
should we get rid of the detections github group? or retain it because there's still a set of people with deeper detection domain knowledge?

Reply:
you're right, I'll remove them